contextMenu.js

```diff
@@ -135,6 +136,26 @@
                             }
                         }
 
+						//update ico
+                        if (ico) {
```

Comment:
While create its inside else if, but here its new if. This changes which one will be used among img/ico on create and update. Can you fix that. 
And if ico was defined on create and later on update is defined than it will not remove the span element.